### PR TITLE
[develop] Integrate Unified Workflow tool set_config

### LINKED ISSUE #829: Surface Climo files not added to namelist correctly
## Expected behavior

The files in the FV3_NML_VARNAME_TO_SFC_CLIMO_FIELD_MAPPING section of parm/fixed_files_mapping.yaml should show up in the `namsfc` section of input namelist in the experiment directory.

## Current behavior

When set_fv3nml_sfc_climo_filenames.py is run to update the namelist with the appropriate files names, it mistakenly adds a section to the namelist labeled "namsfc_dict" and leaves the values in the namsfc section empty. Those then default to the low resolution global gridded data set by the model.

## Machines affected

All.

## Steps To Reproduce

1. Generate workflow for any of the fundamenal tests (any tests really).
2. Open up the input.nml in the experiment directory.
3. Check the namsfc and unused namsfc_dict sections to confirm the behavior described above.

## Detailed Description of Fix (optional)

I caught the error when incorporating the Unified Workflow tools into SRW. I will include the fix as a part of the UW addition.

### PR DESCRIPTION
**Pull request by christinaholtNOAA**
## DESCRIPTION OF CHANGES: 

The use of `ush/set_namelist.py` is being replaced by the UW Tool `set_config.py`. The `set_namelist.py` has not been removed as it may still be of use as a command line tool when manually creating namelists from the `parm/FV3.input.yml` file.

All the python files in the ush directory (top level) have technically been made to pass the linter to facilitate easier testing commands. The files that were modified in the introduction of the set_config tool have been thoroughly linted for real. All others have been skipped with hopes that we can get to those later. Nothing changes about the approach to linting, but the burden of keeping up with which files conform or not is now on the file itself, not on the pylint caller.

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## TESTS CONDUCTED: 

- [x] hera.intel
- [ ] orion.intel
- [ ] cheyenne.intel
- [ ] cheyenne.gnu
- [ ] gaea.intel
- [ ] jet.intel
- [ ] wcoss2.intel
- [ ] NOAA Cloud (indicate which platform)
- [ ] Jenkins
- [x] fundamental test suite
- [ ] comprehensive tests (specify *which* if a subset was used)

## DEPENDENCIES:

The hash to workflow-tools will need to be updated once [PR 253 ](https://github.com/ufs-community/workflow-tools/pull/253) in that repo is merged.

## DOCUMENTATION:

I'm not aware of changes that need to be made to docs. Guidance here would be helpful if I've missed something.

## ISSUE: 

Fixes Issue #829.

## CHECKLIST

- [ ] My code follows the style guidelines in the Contributor's Guide
- [ ] I have performed a self-review of my own code using the Code Reviewer's Guide
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes need updates to the documentation. I have made corresponding changes to the documentation
- [ ] My changes do not require updates to the documentation (explain).
- [ ] My changes generate no new warnings
- [ ] New and existing tests pass with my changes
- [ ] Any dependent changes have been merged and published

## CONTRIBUTORS (optional): 

@WeirAE @elcarpenter @j-derrico @fgabelmannjr @venitahagerty

### CODE REVIEW

**Comment by christinaholtNOAA** [comment]
A quick note...the linter and the unittests are going to continue to fail until the update to workflow-tools is merged. Both pass locally in my space.

Actually...let me temporarily point to my branch. (EDIT: That seems to have done it.)

File: ush/set_ozone_param.py
Comment (MichaelLueken) [inline ush/set_ozone_param.py]: Should this be function rather than functio?
```suggestion
The definition of a function that determines whether ozone
```

File: ush/update_input_nml.py
Comment (MichaelLueken) [inline ush/update_input_nml.py]: This line was added twice.
```suggestion
Set AQM-specific namelist settings
```

**Review by MichaelLueken — APPROVED** [review]
@christinaholtNOAA Overall, these changes look good to me!  I noted two minor spelling/grammatical issues, but that was all.

The fundamental and coverage tests were run on Hera Intel and Jet and all successfully passed (please note that this includes the current DT_ATMOS setting of 30 seconds for the RRFS_AK_3km test as well).

**Comment by christinaholtNOAA** [comment]
@MichaelLueken Thank you for your review. I think we are going to hold off for a sprint or two on this integration. We're taking a moment to apply lessons learned from this work, and refactor workflow-tools just a bit for a better end-user experience.

**Pull request closed without merging**
